aiida/orm/nodes/data/array/bands.py

```diff
@@ -1074,6 +985,115 @@ def show_mpl(self, **kwargs):
         """
         exec(*self._exportcontent(fileformat='mpl_singlefile', main_file_name='', **kwargs))  # pylint: disable=exec-used
 
+    def _prepare_gnuplot(self,
```

Comment:
Is there a reason why you moved `_prepare_gnuplot` down in the file?
That makes it harder to spot the changes you made in review.

Reply:
I just move all `prepare_mpl_*` functions together.  `_prepare_gnuplot` was inserted in the middle of these `_mpl*` functions.